public/src/components/calls/side_bar_form.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Row, Col } from 'react-bootstrap';
+import { Field } from 'redux-form';
+
+import FieldGroup from '../../components/common/form/field_group';
+
+const SideBarForm = (props) => {
+  const { handleSubmit } = props;
+  const inputFieldProps = {
+    label: 'notes',
+    name: 'Notes'
```

Comment:
i think these two can be switched (label is capitalized, name is not)

Reply:
✔️